.github/workflows/ci.yml

```
@@ -18,6 +25,12 @@ env:
   MAKEFLAGS: "-j4"
 
 jobs:
+  build-docs:
+    uses: ./.github/workflows/build-docs.yaml
+    with:
+      #deploy: ${{ (github.event_name == 'push' && startsWith(github.ref, 'refs/tags')) || (github.event_name == 'workflow_dispatch' && inputs.deploy-docs == true) }}
+      deploy: true
```

```suggestion
      deploy: ${{ (github.event_name == 'push' && startsWith(github.ref, 'refs/tags')) || (github.event_name == 'workflow_dispatch' && inputs.deploy-docs == true) }}
```

TODO: revert this before merging.

I'm proposing only updating the docs site on the builds triggered by merges to `main`. Just had this set to `true` to test this PR.